front/src/modules/ui/filter-n-sort/states/filtersWhereScopedSelector.ts

```diff
@@ -0,0 +1,14 @@
+import { selectorFamily } from 'recoil';
```

Comment:
selectors should be in /states/selectors subfolder: https://docs.twenty.com/developer/frontend/folder-architecture